packages/react-core/src/components/JumpLinks/JumpLinksItem.tsx

```diff
@@ -38,9 +39,11 @@ export const JumpLinksItem: React.FunctionComponent<JumpLinksItemProps> = ({
       {...(isActive && { 'aria-current': 'location' })}
       {...props}
     >
-      <a className={styles.jumpLinksLink} href={href} onClick={onClick}>
-        <span className={styles.jumpLinksLinkText}>{children}</span>
-      </a>
+      <span className={styles.jumpLinksLink}>
+        <a className={css(buttonStyles.button, buttonStyles.modifiers.link)} href={href} onClick={onClick}>
```

Comment:
Passed the classes in instead of using Button with props just to retain the onClick event type. No strong opinion about this, so I can just update the onClick event type and use the Button component proper here instead if we want.

Reply:
I would update this to use the Button if possible.